packages/@webex/plugin-meetings/src/common/collection.ts

```diff
@@ -13,10 +15,11 @@ export default class Collection {
    * @param {String} property
    * @memberof Collection
    */
-  constructor(property) {
+  constructor(property: string) {
     if (!property || !isString(property)) {
```

Comment:
Typescript only helps only during compile time and we leverage the type info for easing the development. At the end when we compile and bundle our code it won't have any affect of typescript check so, we would need that check here.

Reply:
With my understanding typescript helps while development as well by showing error at that particular line of code for eg, here if we try to create an instance of Collection class by passing a boolean parameter instead of string then it would show this error `Argument of type 'boolean' is not assignable to parameter of type 'string'` which makes developer understand to pass expected type at the development time only. Please correct me if I understood it wrong.

Reply:
<img width="1050" alt="typescript" src="https://user-images.githubusercontent.com/16598065/203616361-9a5f0a96-5395-4ecf-be69-34abcce6f74c.png">

We can discuss 1-1 if you want more info.